prow/cmd/jenkins-operator/main.go

```diff
@@ -99,6 +102,9 @@ func main() {
 		ghc = github.NewClient(oauthSecret, *githubEndpoint)
 	}
 
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, os.Interrupt, os.Kill, syscall.SIGTERM)
```

Comment:
The kill signal cannot be received by a process, the kernel just kills the process so you don't need to include it here or in plank.

Reply:
ack